controller/formation.go

```diff
@@ -303,7 +318,14 @@ func (r *FormationRepo) unsubscribeLocked(ch chan *ct.ExpandedFormation) {
 		for range ch {
 		}
 	}()
+	si, ok := r.subscriptions[ch]
 	delete(r.subscriptions, ch)
+	if ok {
+		si.uc <- struct{}{}
+		si.wg.Wait()
```

Comment:
But that way we are **not ALSO selecting** on `stopCh` inside `sendUpdatedSince`. Right?